google/test-fixtures/cloudfunctions/firestore_trigger.js

```diff
@@ -7,7 +7,7 @@
 exports.helloFirestore = function (event, callback) {
     const messageId = event.params.messageId;
 
-    console.log(`Received message ${messageId}`);
+     console.log(`Received message ${messageId}`);
```

Comment:
This shouldn't be in here, but I'll go ahead and merge this and then make sure it gets fixed.